### ISSUE TITLE
Licence issue

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#258]
**Reported by misterx on 4 Jul 2007 20:05 UTC**

GPL and Openssl licence doesn't mix very well.

There should be an exception like the following if we still want to use it:
```
In addition, as a special exception, the copyright holders give
permission to link the code of portions of this program with the OpenSSL
library.
You must obey the GNU General Public License in all respects for all of
the code used other than OpenSSL. If you modify file(s) with this
exception, you may extend this exception to your version of the file(s),
but you are not obligated to do so. If you do not wish to do so, delete
this exception statement from your version. If you delete this exception
statement from all source files in the program, then also delete it here.
```

That point must be done before any release of 1.0

**Comment by aircrack-ng** [comment]
**Comment by misterx on 4 Jul 2007 20:05 UTC**

http://lists.debian.org/debian-legal/2004/05/msg00595.html:

1. Add the attached LICENSE file to your code. This file tells that your
code is GPL, with an exception which allows linking with OpenSSL. Here
is the text of the exception:

```
***
  In addition, as a special exception, the copyright holders give
  permission to link the code of portions of this program with the
  OpenSSL library under certain conditions as described in each
  individual source file, and distribute linked combinations
  including the two.
  You must obey the GNU General Public License in all respects
  for all of the code used other than OpenSSL.  If you modify
  file(s) with this exception, you may extend this exception to your
  version of the file(s), but you are not obligated to do so.  If you
  do not wish to do so, delete this exception statement from your
  version.  If you delete this exception statement from all source
  files in the program, then also delete it here.
***
```

2. Add to the beginning of all of your code source files this exception
statement.

3. Add the attached LICENSE.OpenSSL to your source code.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 4 Jul 2007 20:05 UTC**

(In #757) Added OpenSSL licence exception (Closes: #258).
